scripts/src/java/org/oppia/android/scripts/coverage/CoverageRunner.kt

```diff
@@ -56,13 +61,18 @@ class CoverageRunner(
     val sfStartIdx = coverageData.indexOfFirst {
       it.startsWith("SF:") && it.substringAfter("SF:").substringAfterLast("/") == extractedFileName
     }
-    require(sfStartIdx != -1) {
-      "Coverage data not found for the file: $extractedFileName"
-    }
+    if (sfStartIdx == -1) return generateFailedCoverageReport(
```

Comment:
Here & below: please add braces for the statement.

Reply:
Done. Added them.